controllers/replica_set_controller.go

```diff
@@ -559,6 +562,29 @@ func buildAutomationConfig(mdb mdbv1.MongoDBCommunity, auth automationconfig.Aut
 		Build()
 }
 
+func guessEnterprise(mdb mdbv1.MongoDBCommunity) bool {
```

Comment:
On enterprise we will need something similar. We could make this re-usable by changing the signature to take a wrapper of containers instead

Reply:
I think guessing is not necessary in the enterprise, where we rely only on the enterprise server image. So I believe we'll always set `true` there.

Reply:
This has been answered internally.